[Bug Report] Cannot find module '../el-scrollbar/src/util' in Server-Side Rendering

<!-- generated by https://elementui.github.io/issue-generator DO NOT REMOVE -->

### Element Plus version
1.0.2-beta.48

### OS/Browsers version
Windows 10, Chrome 91.0.4472.114

### Vue version
3.1.1

### Reproduction Link
https://codesandbox.io/s/102-beta48-el-scrollbar-ssr-bug-787w1?file=/src/pages/Home.vue

### Steps to reproduce
1. use `el-scrollbar` tag in template
2. run command `yarn ssr`

### What is Expected?
node server runs successfully

### What is actually happening?
```
internal/modules/cjs/loader.js:888
  throw err;
  ^

Error: Cannot find module '../el-scrollbar/src/util'
Require stack:
- [Project_Path]\node_modules\element-plus\lib\el-virtual-list\index.js
- [Project_Path]\node_modules\element-plus\lib\el-select-v2\index.js
- [Project_Path]\node_modules\element-plus\lib\index.js
- [Project_Path]\dist\server\js\app.0595415c.js
- [Project_Path]\server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
    at Function.Module._load (internal/modules/cjs/loader.js:730:27)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> ([Project_Path]\node_modules\element-plus\lib\el-virtual-list\index.js:11:14)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '[Project_Path]\\node_modules\\element-plus\\lib\\el-virtual-list\\index.js',
    '[Project_Path]\\node_modules\\element-plus\\lib\\el-select-v2\\index.js',
    '[Project_Path]\\node_modules\\element-plus\\lib\\index.js',
    '[Project_Path]\\dist\\server\\js\\app.0595415c.js',
    '[Project_Path]\\server.js'
  ]
}
error Command failed with exit code 1.
```
<!-- generated by https://elementui.github.io/issue-generator DO NOT REMOVE -->

--- a/packages/virtual-list/src/components/scrollbar.ts
+++ b/packages/virtual-list/src/components/scrollbar.ts
@@ -1,11 +1,11 @@
 import { computed, defineComponent, ref, reactive, onMounted, onBeforeUnmount, watch } from 'vue'
-import { BAR_MAP } from '@element-plus/scrollbar/src/util'
 import { on, off } from '@element-plus/utils/dom'
 import { rAF, cAF } from '@element-plus/utils/raf'
 import isServer from '@element-plus/utils/isServer'
 
 import { DefaultScrollBarProps, ScrollbarSizeKey, ScrollbarDirKey, SCROLLBAR_MIN_SIZE } from '../defaults'
 import { renderThumbStyle } from '../utils'
+import { BAR_MAP } from '../../../scrollbar/src/util'
 
 import type { CSSProperties } from 'vue'
 

